packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -219,8 +218,9 @@ export const formatters: Formatters = {
       }
 
       if (event.session_start && event.session_stop) {
-        const duration = moment(event.session_stop).diff(event.session_start);
-        const durationText = moment.duration(duration).humanize();
+        const start = new Date(event.session_start);
+        const end = new Date(event.session_stop);
```

Comment:
should this be converted in `makeRecording`?

Reply:
no because `makeRecording` is only used for the `Recordings` tab, not audit log. Everything else in `makeEvent` deals with raw response data.